hubspot.com.hubspot-template-3.json

```diff
@@ -3,8 +3,9 @@
     "providerName": "HubSpot",
     "serviceId": "hubspot-template-3",
     "serviceName": "HubSpot CMS Site",
+    "syncPubKeyDomain": "domainconnect.hubspot.com",
+    "syncRedirectDomain": "app.hubspot.com, app.hubspotqa.com, api.hubspot.com, api.hubspotqa.com",
     "version": 1,
```

Comment:
One more thing, please bump the version. Sorry I did not notice that before.

Reply:
Done :)